.github/workflows/test-action.yml

```diff
@@ -0,0 +1,50 @@
+name: Smoke test the action
```

Comment:
Consider adding substantially more info here, maybe even a README in the directory?

The testing setup is non-trivial, and it's hard to understand which piece is testing what. For example, it isn't clear why `test-action` and `test-inputs` are separate workflow files. If someone were adding a new test, it isn't clear which file it should go in.

Reply:
Definitely a good idea, added a README. I also made two other changes that you suggested, please have a look now.